spel/minimal-linux.json

```diff
@@ -361,6 +458,20 @@
             "source": "/tmp/manifest.log",
             "destination": ".spel/{{ user `spel_version` }}/{{ user `spel_identifier` }}-{{ build_name }}.manifest",
             "direction": "download"
+        },
+        {
+            "type": "shell",
+            "only": [
+                "minimal-centos-7.4-azure-vhd",
+                "minimal-centos-7.4-azure-image"
+            ],
+            "execute_command": "chmod +x {{ .Path }}; {{ .Vars }} sudo -E sh '{{ .Path }}'",
+            "inline": [
+              "chkconfig waagent on",
+              "/usr/sbin/waagent -force -deprovision && export HISTSIZE=0 && sync"
+            ],
+            "inline_shebang": "/bin/sh -x",
```

Comment:
You can use `execute_command` above to pass the `-x` flag, instead of injecting it into the shebang:

```
"execute_command": "chmod +x {{ .Path }}; {{ .Vars }} sudo -E sh -x '{{ .Path }}'"
```

And maybe use `-ex` to fail the build if one of the commands happens to fail...

Reply:
Was initially trying to stick to the packer example provided (https://www.packer.io/docs/builders/azure.html#linux) but now see the pattern elsewhere in minimal-linux.json.  (testing updates before updating commit)